src/loaders/compute_tools/microtrait/microtrait.py

```diff
@@ -262,6 +272,22 @@ def _run_microtrait(
     create_jsonl_files(genome_dir / MICROTRAIT_CELLS, cells_meta)
     create_jsonl_files(genome_dir / MICROTRAIT_DATA, heatmap_row)
 
+    end_time = time.time()
+    run_time = end_time - start
+    print(
+        f'Used {round(run_time / 60, 2)} minutes to execute Microtrait for {data_id}')
+
+    # Save run info to a metadata file in the output directory for parsing later
+    metadata = {'source_file': str(fna_file),
+                'data_id': data_id,
+                'tool_name': 'microtrait',
+                'version': 'None',
```

Comment:
There must be at least a git hash right

Reply:
👍 I updated to `ukaraoz/microtrait - release tag kb`. Do you think that makes sense?

Reply:
Seems like the best we can do, although maybe you should split it into git_url and release_tag?

Reply:
👍

Reply:
:+1: